hibernate-orm-quickstart/src/main/java/org/acme/hibernate/orm/FruitResource.java

```diff
@@ -43,7 +41,7 @@
 
     @GET
     @Path("{id}")
-    public Fruit getSingle(@PathParam Integer id) {
+    public Fruit getSingle(@PathParam("id") Integer id) {
```

Comment:
You don't have the equivalent in RESTEasy Reactive to avoid having the name?

Reply:
You basically don't need any annotation at all :)